lib/setupmanagers/hckclient.rb

```diff
@@ -76,7 +76,7 @@ def run_pre_test_commands
       @project.engine.drivers&.each do |driver|
         driver.pretestcommands&.each do |command|
```

Comment:
in this commit you renamed `pretestcommands -> post_start_commands` in JSON but `lib/models/driver.rb` is not updated. Also `def run_pre_test_commands` is not updated. So you have broken AutoHCK after applying this commit.